packages/kit/src/views/Market/MarketDetailV2/components/InformationTabs/components/TransactionsHistory/hooks/useTransactionItemData.ts

```
@@ -0,0 +1,74 @@
+
+  const typeText = isBuy
+    ? intl.formatMessage({
+        id: ETranslations.dexmarket_details_transactions_buy,
+      })
+    : intl.formatMessage({
+        id: ETranslations.dexmarket_details_transactions_sell,
+      });
+
+  const price = isBuy ? item.to.price : item.from.price;
+  const value = (
+    parseFloat(item.from.amount) * parseFloat(item.from.price)
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)